### ISSUE TITLE
Expose a way to change the host in config

### ISSUE THREAD

**Issue by christianlime** [facebook/metro#1389]
**Do you want to request a *feature* or report a *bug*?**
Feature

**What is the current behavior?**
I can only customize port with `localhost`, `tunnel` or `lan`

**What is the expected behavior?**
I want to be able to provide an alias for `localhost` for local web development. I don't want to be limited to only use `localhost:{port}`. This would be similar to using [server.host](https://vite.dev/config/server-options#server-host) as vite does, or webpack does with [devServer.host](https://webpack.js.org/configuration/dev-server/#devserverhost)

**Comment by robhogan** [comment]
Yeah, that's a reasonable request - it shouldn't be too difficult to implement following the example of the way `port` is passed through from both config and CLI args, we'd welcome a PR.

**Comment by 0xSaksham** [comment]
is this issue completed?
